src/main.c

```diff
@@ -373,28 +373,28 @@ void WriteInterfaces()
         switch (Interface->Type) {
         case PCAPNG_LINKTYPE_ETHERNET:
             if (Interface->IsVMNic) {
-                printf("IF: medium=%s\tID=%u\tIfIndex=%u\tVlanID=%i",
+                printf("IF: medium=vNIC-%s\tID=%u\tIfIndex=%u\tVlanID=%i",
                     Interface->VMNic.SourceNicType,
                     Interface->PcapNgIfIndex,
-                    Interface->VMNic.SourcePortId,
+                    Interface->LowerIfIndex,
                     Interface->VlanId
                 );
                 StringCchPrintfA(
                     IfName,
                     IF_STRING_MAX_SIZE,
-                    "%s:%s:%lu:%i",
-                    Interface->VMNic.SourcePortName,
+                    "vNIC:%s:%s:%lu:%i",
                     Interface->VMNic.SourceNicType,
-                    Interface->VMNic.SourcePortId,
+                    Interface->VMNic.SourcePortName,
+                    Interface->LowerIfIndex,
                     Interface->VlanId
                 );
             } else {
-                printf("IF: medium=eth\tID=%u\tIfIndex=%u\tVlanID=%i", Interface->PcapNgIfIndex, Interface->LowerIfIndex, Interface->VlanId);
+                printf("IF: medium=eth\t\t\tID=%u\tIfIndex=%u\tVlanID=%i", Interface->PcapNgIfIndex, Interface->LowerIfIndex, Interface->VlanId);
```

Comment:
Why was this tab changed to three tabs? Can you share the before and after appearance?

Reply:
This is to align output as the VMSwitch interface name are longer (prefix vNIC-XXXX)

After :
IF: medium=eth                         ID=0    IfIndex=3       VlanID=0
IF: medium=eth                         ID=1    IfIndex=5       VlanID=0
IF: medium=vNIC-External        ID=2    IfIndex=28      VlanID=98       (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=3    IfIndex=28      VlanID=399      (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=4    IfIndex=28      VlanID=298      (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=5    IfIndex=28      VlanID=397      (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=6    IfIndex=28      VlanID=97       (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=7    IfIndex=28      VlanID=300      (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=8    IfIndex=28      VlanID=100      (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=9    IfIndex=28      VlanID=0        (LWF over IfIndex 6)
IF: medium=vNIC-External        ID=10   IfIndex=28      VlanID=197      (LWF over IfIndex 6)

Before : 
IF: medium=eth  ID=0    IfIndex=3       VlanID=0
IF: medium=eth  ID=1    IfIndex=5       VlanID=0
IF: medium=External     ID=2    IfIndex=1       VlanID=98       (LWF over IfIndex 6)
IF: medium=External     ID=3    IfIndex=1       VlanID=399      (LWF over IfIndex 6)
IF: medium=External     ID=4    IfIndex=1       VlanID=298      (LWF over IfIndex 6)
IF: medium=External     ID=5    IfIndex=1       VlanID=397      (LWF over IfIndex 6)
IF: medium=External     ID=6    IfIndex=1       VlanID=97       (LWF over IfIndex 6)
IF: medium=External     ID=7    IfIndex=1       VlanID=300      (LWF over IfIndex 6)
IF: medium=External     ID=8    IfIndex=1       VlanID=100      (LWF over IfIndex 6)
IF: medium=External     ID=9    IfIndex=1       VlanID=0        (LWF over IfIndex 6)
IF: medium=External     ID=10   IfIndex=1       VlanID=197      (LWF over IfIndex 6)
IF: medium=External     ID=11   IfIndex=1       VlanID=398      (LWF over IfIndex 6)
IF: medium=Synthetic    ID=12   IfIndex=2       VlanID=0        (LWF over IfIndex 6)
IF: medium=External     ID=13   IfIndex=1       VlanID=297      (LWF over IfIndex 6)

Reply:
OK.

Reply:
I can see this is not aligned in this window, but well in the console output!